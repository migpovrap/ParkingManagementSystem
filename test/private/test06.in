p xTmNqP2eCNhm 6 0.35 0.75 33.00
p hx8Ej0GwjrLsqevXyw 4 0.70 0.90 39.60
p FdlUFOXMyTGDUbNUz 5 0.60 0.65 28.60
p
e xTmNqP2eCNhm HW-MW-42 31-03-2024 00:00
e hx8Ej0GwjrLsqevXyw 42-87-MO 31-03-2024 00:27
e FdlUFOXMyTGDUbNUz 01-18-DZ 31-03-2024 00:39
e FdlUFOXMyTGDUbNUz NT-ZP-76 31-03-2024 00:44
e xTmNqP2eCNhm 57-43-KR 31-03-2024 01:00
e hx8Ej0GwjrLsqevXyw SQ-38-KA 31-03-2024 01:15
e FdlUFOXMyTGDUbNUz 07-EL-86 31-03-2024 01:33
e xTmNqP2eCNhm 71-BL-63 31-03-2024 01:35
e FdlUFOXMyTGDUbNUz 88-41-VX 31-03-2024 02:00
e xTmNqP2eCNhm PQ-26-XR 31-03-2024 02:12
e FdlUFOXMyTGDUbNUz LY-25-41 31-03-2024 02:18
e xTmNqP2eCNhm QG-26-IG 31-03-2024 02:54
e hx8Ej0GwjrLsqevXyw 62-WS-LZ 31-03-2024 03:00
e hx8Ej0GwjrLsqevXyw WB-XP-11 31-03-2024 03:20
e xTmNqP2eCNhm 54-OL-32 31-03-2024 03:48
p
q
