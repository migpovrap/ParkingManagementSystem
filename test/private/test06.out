xTmNqP2eCNhm 6 6
hx8Ej0GwjrLsqevXyw 4 4
FdlUFOXMyTGDUbNUz 5 5
xTmNqP2eCNhm 5
hx8Ej0GwjrLsqevXyw 3
FdlUFOXMyTGDUbNUz 4
FdlUFOXMyTGDUbNUz 3
xTmNqP2eCNhm 4
hx8Ej0GwjrLsqevXyw 2
FdlUFOXMyTGDUbNUz 2
xTmNqP2eCNhm 3
FdlUFOXMyTGDUbNUz 1
xTmNqP2eCNhm 2
FdlUFOXMyTGDUbNUz 0
xTmNqP2eCNhm 1
hx8Ej0GwjrLsqevXyw 1
hx8Ej0GwjrLsqevXyw 0
xTmNqP2eCNhm 0
xTmNqP2eCNhm 6 0
hx8Ej0GwjrLsqevXyw 4 0
FdlUFOXMyTGDUbNUz 5 0
